ToGather/ToGather/Global/Service/FirebaseManager.swift

```diff
@@ -9,9 +9,11 @@ import Firebase
 import SwiftUI
 
 final class FirebaseManager: ObservableObject {
-    @EnvironmentObject var onboardingViewModel: UserViewModel
+//    @EnvironmentObject var userViewModel: UserViewModel
     @Published var nicknameArray: [String] = []
-    
+    @Published var notification = [Notification]()
+    @Published var userViewModel = UserViewModel()
+
     static let shared = FirebaseManager()
     
```

Comment:
싱글톤으로 만들려면 
```suggestion
    private init() {}
```
을 추가해야 외부에서 객체를 생성하는 것을 막을 수 있습니다!

Reply:
감사합니다!!